File: app/src/main/java/corp/uoc/edu/MainActivity.java
package corp.uoc.edu;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        int mat = 5;
        int quim = 5;
        int fis = 5;
        int prom = 0;

        prom = (mat + quim + fis) / 3;

        if(prom >   = 6) {
            Toast.makeText(this, "Aprovat", Toast.LENGTH_SHORT).show();
        } else if (prom <=5) {
            Toast.makeText(this, "Suspès", Toast.LENGTH_SHORT).show();
        }
    }
}

File: app/src/main/java/corp/uoc/edu/MainActivity.kt
package corp.uoc.edu

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.content.Intent
import android.util.Log
import android.widget.Button
import corp.uoc.edu.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)

        binding.button.setOnClickListener {
            val i = Intent(applicationContext, SecondActivity::class.java)
            startActivity(i)
        }
        Log.i("MainActivity", "Jordi Pujol")
    }
}